npm/private/npm_import.bzl

```diff
@@ -1148,6 +1158,16 @@ def npm_import(
 
         dev: Whether this npm package is a dev dependency
 
+        exclude_patterns: List of glob patterns to exclude from the linked package.
```

Comment:
API naming:
* `exclude_package_contents` (from @thesayyn, this is my preference atm...)
* `excludes`
* `package_excludes`
* ...?

Then the `npm_package_store` API would just drop the `s` at the end.

@gregmagolan @alexeagle @thesayyn opinions?

Reply:
I'm fine with Sahin's preference